server/src/main/java/com/linecorp/centraldogma/server/metadata/PerRolePermissions.java

```diff
@@ -98,16 +98,23 @@ public static PerRolePermissions ofPrivate() {
      */
     private final Set<Permission> guest;
 
+    /**
+     * {@link Permission}s for a {@link ProjectRole#ANONYMOUS}.
+     */
+    private final Set<Permission> anonymous;
+
     /**
      * Creates an instance.
      */
     @JsonCreator
     public PerRolePermissions(@JsonProperty("owner") Iterable<Permission> owner,
                               @JsonProperty("member") Iterable<Permission> member,
-                              @JsonProperty("guest") Iterable<Permission> guest) {
+                              @JsonProperty("guest") Iterable<Permission> guest,
+                              @JsonProperty("anonymous") Iterable<Permission> anonymous) {
         this.owner = Sets.immutableEnumSet(requireNonNull(owner, "owner"));
         this.member = Sets.immutableEnumSet(requireNonNull(member, "member"));
         this.guest = Sets.immutableEnumSet(requireNonNull(guest, "guest"));
+        this.anonymous = Sets.immutableEnumSet(requireNonNull(anonymous, "anonymous"));
```

Comment:
Is this compatible with the existing data? `anonymous` may be null if `PerRolePermissions` is serialized like:
```json
"perRolePermissions": {
  "owner": [
    "READ",
    "WRITE"
  ],
  "member": [
    "READ",
    "WRITE"
  ],
  "guest": [
    "READ",
    "WRITE"
  ]
}
```

Reply:
Thanks! Addressed. 😉

Reply:
Oh, how was it addressed 🤔 ?

Reply:
Here it is 😉 : https://github.com/line/centraldogma/pull/917/commits/bb55dcabc6701dddb633c8c6a48b81eee7860f9b